pootle/templates/user/profile.html

```diff
@@ -18,8 +18,8 @@ <h1 class="user-name">
         {{ object.display_name }}
       {% endif %}
     </h1>
-    {% if user.is_superuser or request.permissions.administrate %}
-    <p><a href="mailto:{{ object.email }}">{{ object.email }}</a></p>
+    {% if user.is_superuser or user.is_manager %}
```

Comment:
my only concern here is that we dont want to make this query more than once in a request - ideally we could cache it - or alternatively we could set it in the view context